### ISSUE TITLE
Page plan details does not need two save buttons

### ISSUE THREAD

**Issue by nicolasfranck** [DMPRoadmap/roadmap#2956]
Please complete the following fields as applicable:

**What version of the DMPRoadmap code are you running? (e.g. v2.2.0)**

`3.0.2`

**Behaviour:**

The page `/plans/:id` (project details) has two save buttons,
one in the left below, and one on the right below.
Both are part of the same HTML form and therefore
send to the same controller.

Keep one? Of course I can imagine that this second button
on the right was added because some users didn't find the
save button (long texts). Or put that one save button on top? That
last idea requires some rethinking of course of how other
forms are organised.

**Comment by briri** [comment]
Yes, this was added due to user's missing the Save button on the left. It is not optimal.

related to #2945, #1166 and #2589 which involve a rethinking of how the guidance selection is managed. For example if me move the guidance selection off of the Project Details page we could remove the 2nd button.

**Comment by nicolasfranck** [comment]
Maybe open a modal where you have all the room available to show this tree, and add a save button there? That would save the other too of course
